src/routes/Project/components/Table.js

```diff
@@ -193,9 +203,10 @@ export default class Table extends Component {
   render() {
     const METHOD_NAME = 'render';
 
-    // $log.d(CLASS_NAME, METHOD_NAME, 'Rendering');
+    let gridWidth = this.props.numMeasures * this.props.timeInterval;
+
     return (
-      <div id="table-component">
+      <div id="table-component" style={{width: this.props.widthPx, backgroundSize: `${gridWidth}px`}}np>
```

Comment:
that np is definitely a typo. table.scss has the magic css to make grid lines, background-size sets the spacing of those lines